changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.1.0 - xxxx-xx-xx =
+* Fix - Remove `redirect_url` parameter from Express Checkout payment flow
```

Comment:
```suggestion
* Update - Remove `redirect_url` parameter from ECE payment flow
```

Reply:
🤦 I applied it and used GH comments to apply it to the `readme.txt` file. But instead of copying from `changelog.txt` into `readme.txt`, I did the opposite.
Fixing in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4776